gnmi/gnmi_test.go

```diff
@@ -909,8 +867,32 @@ func TestSetYGNMI(t *testing.T) {
 			if d := errdiff.Check(err, tt.wantErr); d != "" {
 				t.Errorf("Set() unexpected err: %s", d)
 			}
-			if tt.wantErr == "" {
-				tt.checkOp(t, c)
+			if tt.wantErr != "" {
+				return
+			}
+
+			got, ok := tt.getOp(t, c)
+			switch want := tt.wantValue.(type) {
+			case nil:
+				if ok {
+					t.Errorf("Got present, want not present")
+				}
+			case ygot.GoStruct:
```

Comment:
is there a benefit to doing ygot.Diff vs cmp.DIff here?

Reply:
cmp.Diff doesn't work because it treats nil vs. empty structures as different.